CONTRIBUTING.md

```diff
@@ -36,24 +36,7 @@ at least a maintainer of the platform (a maintainer making a PR themselves count
 
 ## Maintainers & Testers
 
-Winit is managed by several people, each with their specialities, and each maintaining a subset of the
-backends of winit. As such, depending on your platform of interest, your contacts will be different.
-
-This table summarizes who can be contacted in which case, with the following legend:
-
-- `M` - Maintainer: is a main maintainer for this platform
-- `C` - Collaborator: can review code and address issues on this platform
-- `T` - Tester: has the ability of testing the platform
-- ` `: knows nothing of this platform
-
-| Platform            | Windows | macOS | X11   | Wayland | Android | iOS   | Emscripten |
-| :---                | :---:   | :---: | :---: | :---:   | :---:   | :---: | :---:      |
-| [@mitchmindtree]    | T       |       | T     | T       |         |       |            |
-| [@Osspial]          | M       |       | T     | T       | T       |       | T          |
-| [@vberger]          |         |       | T     | M       |         |       |            |
-| [@mtak-]            |         | T     |       |         | T       | M     |            |
-
-[@mitchmindtree]: https://github.com/mitchmindtree
-[@Osspial]: https://github.com/Osspial
-[@vberger]: https://github.com/vberger
-[@mtak-]: https://github.com/mtak-
+Please refer to [this table in the Wiki](https://github.com/rust-windowing/winit/wiki/Testers-and-Contributors)
+for the list of active contributors and testers.
```

Comment:
I suggest to reword this as:

The [current list of testers and contributors](https://github.com/rust-windowing/winit/wiki/Testers-and-Contributors) can be found on the Wiki.

```markdown
The [current list of testers and contributors](https://github.com/rust-windowing/winit/wiki/Testers-and-Contributors) can be found on the Wiki.
```